packages/components/dialog/src/dialog.vue

```diff
@@ -62,25 +62,18 @@ import { dialogProps, dialogEmits } from './dialog'
 import { elDialogInjectionKey } from './token'
 import { useDialog } from './use-dialog'
 
-import type { SetupContext, Ref } from 'vue'
-import type { DialogEmits } from './dialog'
-
 defineOptions({
   name: 'ElDialog',
 })
 
+defineEmits(dialogEmits)
```

Comment:
Why to move order

Reply:
Please move to line 71
```suggestion
const emit = defineEmits(dialogEmits)
```